docs(flypad-translations): add localazy logo and partnership information (#770)
flybywiresim/docs: docs/dev-corner/dev-guide/specific/flypad-translations.md

This page is pulled externally from the A32NX repository and describes the processes involved with adding features to the EFB and sourcing translations.

> We're thrilled to be partnered with Localazy, who offers top-notch software localization services bringing multiple language support to our EFB enhancing the user experience.

Reviewer: Unlike a marketing announcement the docs should be written in a timeless manner in my opinion. 
I'll make a suggestion later tonight also with some grammar corrections.

> We're thrilled to be partnered with Localazy, who offers top-notch software localization services bringing multiple language support to our EFB enhancing the user experience.

Reviewer: ```suggestion
We are partnering with Localazy, who offer top-notch software localization services, bringing multi-language support to our EFB, enhancing the user experience.
```

Author: Would it be better to keep it "partnered"? Otherwise this would be good.

Reviewer: "We have partnered" would be even better.

Author: Done deal! Thanks Slein
